vignettes/decorate-module-output.Rmd

```diff
@@ -43,22 +48,23 @@ It is recommended to review the module documentation or source code to understan
 
```

Comment:
On the first point under Requirements and Limitations, should we be more explicit on how a user could check whether a module is designed to work with decorators or not?

Reply:
I've added a link to "include-decorators-in-a-teal-module" section